### ISSUE TITLE
Connect AngularFire with Angular 🔥

### ISSUE THREAD

**Issue by jdjuan** [angular-medellin/form#6]
Let's write a proof-of-concept using Angular, Firebase and AngularFire.

Once #4 is finished, clone the repo and start working on it.

The POC should contain:
- A button that once pressed it pushes a new registry to the Firebase database.

PS: The Firebase database can be one of yours in the meanwhile.

**Comment by Shinkei** [comment]
@ Angular-medellin / form-team I have already updated the proof of concept code to the branch [firebasePOC](https://github.com/angular-medellin/form/tree/firebasePOC), there I used firebase and Angularfire2 to list, retrieve and add "characters" 👨 (just because) I know it is not related with the Form but remember it is a POC. I also add some firebase authentication with the google account (I followed a tutorial 📖 ).
I will continue working on the app to complete all functionalities and change the components to fit the project form, but I do not know if you want a PR to check it till now
I will be looking forward
Thanks

**Comment by jdjuan** [comment]
Hey that's awesome @Shinkei ! Good Job! 👍 

Let's make a PR to see how it looks so far! 👌 

Our first MVP will use MailChimp to hold the information of our users, at some point we will migrate your  efforts so we can build a more robust and feature-rich application!

Thank you again! Hope to see that PR soon!

**Comment by Shinkei** [comment]
Nice, Mailchimp is a good option,
So in this case,  @jdjuan do you want me to continue working on this feature or maybe help with another issue?

**Comment by jdjuan** [comment]
Firebase is a great ally! I hope to see it more in our meetups as time goes by, if you want to keep tinkering with Firebase that'd be just fine! 😁 In fact, maybe you can share your XP with a talk, how about that? 🤙 Would you like?

**Comment by Shinkei** [comment]
hehehehe I think I can, that would be a good experience, I am not a peoples person, but maybe you could help me to do it well :pray: 
BTW, if we don't use it in the form, maybe the firebase could be useful for the blog

**Comment by jdjuan** [comment]
Hey that's right! @Shinkei we can definitely take advantage of it in other projects like the Blog! We haven't discussed implementation yet, but when we do, I will make sure you be there! 👌 

I can help you out for sure! It's gonna be an interesting challenge, why don't you start creating an issue here: https://github.com/angular-medellin/meetup

**Comment by Shinkei** [comment]
I close this issue due to we are nog going to use firebase
